integration/setup/startup.go

```diff
@@ -47,11 +47,12 @@ var startupWG sync.WaitGroup
 // Startup initializes things that should be initialized only once.
 func Startup() {
 	opts := &logging.NewHandlerOpts{
-		Base:          "console",
-		Level:         slog.LevelDebug,
-		RemoveTime:    true,
-		RemoveLevel:   true,
-		CheckMessages: false, // TODO https://github.com/FerretDB/FerretDB/issues/4511
+		Base:        "console",
+		Level:       slog.LevelDebug,
+		RemoveTime:  true,
+		RemoveLevel: true,
+		// testify produces incompatible log messages
+		CheckMessages: false,
```

Comment:
Something strange is happening: we can't set `CheckMessages` because of testify…